lightning-invoice/src/lib.rs

```diff
@@ -72,6 +108,7 @@ pub const DEFAULT_EXPIRY_TIME: u64 = 3600;
 /// [`MIN_FINAL_CLTV_EXPIRY`]: lightning::ln::channelmanager::MIN_FINAL_CLTV_EXPIRY
 pub const DEFAULT_MIN_FINAL_CLTV_EXPIRY: u64 = 18;
 
+#[cfg(feature = "std")]
```

Comment:
nit: Place below the docs for consistency and ease of association in the case of long docs. Similar elsewhere.

Reply:
:+1: